docs/design/README.md

```diff
@@ -967,6 +997,9 @@ the assignment.
 Unlike C++, these assignments are statements, not expressions, and don't return
 a value.
 
+> **Note:** The semantics of assignment are provisional. See pending proposal
+> [#821: Values, variables, pointers, and references](https://github.com/carbon-language/carbon-lang/pull/821).
+
```

Comment:
Not sure it even makes sense to include #821 vs. just marking as provisional?

Reply:
It leaves a breadcrumb to see where the content from the README came from, and where to go to read more.